e3sm_to_cmip/util.py

```diff
@@ -1051,10 +1051,14 @@ def precheck(inpath, precheck_path, variables, realm):
     start, end = get_years_from_raw(inpath, realm, variables[0])
     var_map = [{"found": False, "name": var} for var in variables]
 
+    logger.info(f"precheck: working on year-range {start} to {end}")
```

Comment:
These logger lines have been adopted to user `logger` now

Reply:
Excellent.  Hope I didn't leave any behind.